homeassistant/components/matter/lock.py

```diff
@@ -114,6 +120,9 @@ async def async_unlock(self, **kwargs: Any) -> None:
 
     async def async_open(self, **kwargs: Any) -> None:
         """Open the door latch."""
+        # optimistically signal unlocking to state machine
+        self._attr_is_unlocking = True
```

Comment:
Should we use `_attr_is_opening`?

Reply:
Do we have that?

Reply:
https://github.com/home-assistant/core/blob/c3830a58cc89a2b26870731dbdcf48b8f9d450e3/homeassistant/components/lock/__init__.py#L213-L216

https://github.com/home-assistant/core/blob/c3830a58cc89a2b26870731dbdcf48b8f9d450e3/homeassistant/components/lock/__init__.py#L272-L279

https://developers.home-assistant.io/docs/core/entity/cover#properties